### PR TITLE
Upgraded storybook to v5

### PR DESCRIPTION
**Pull request by eswarasai**
### Description
- [x] Upgraded `storybook` to v5
- [x] Resolved issues with getting initial setup running due to below error
```
Error: Cannot find module 'babel-loader/package.json' from 'status-components'
```

### CODE REVIEW

**Comment by eswarasai** [comment]
@jeluard -- I see that the project has just been created and would love to setup the initial architecture for creating `storybook` components for respective `Status.im` UI components.

Before getting started with the development of actual components, I thought it'd be better to setup a basic architecture in place especially linting, CI/CD and `react-native` support. Let me know your thoughts on this so that I can take up #5 and #3 in the order before proceeding to #2. Thanks!

**Comment by jeluard** [comment]
@eswarasai Most definitely!
Can you share your plan with regard to `react-native`? Ideally components would have a similar syntax gfor both `react` and `react-native`.

File: package.json [hunk 1]
```diff
@@ -3,12 +3,13 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@material-ui/core": "^3.9.2",
```
Comment (jeluard) [inline package.json]: Is this needed? I'd rather not depend on material-ui as status components are significantly different

Comment (eswarasai) [reply package.json]: It was being used in the components `Welcome.jsx` and `Tooltip.jsx`. Hence added. I can definitely remove it.

Comment (jeluard) [reply package.json]: Please! Those components are not used, just copy pasta leftovers :D

Comment (eswarasai) [reply package.json]: Sure 👍

**Comment by eswarasai** [comment]
@jeluard -- I'm going through few example projects for `react-native` in `storybook`. Let me know what you think of this project setup -- https://github.com/wix/react-native-storybook-example

Ideally it makes sense to keep storybook for both React and React-Native separate repos. But since the components can be heavily re-used, I'd recommend going with a monorepo setup with packages for `react-storybook`, `react-native-storybook` and `components`. Does this makes sense?

Let me know your thoughts on this and also feel free to add anything that I'm missing. Thanks!

**Comment by jeluard** [comment]
As much as possible it would be great to have everything in a single source tree. Even sharing code if that's feasible.
 Something like https://github.com/necolas/react-native-web could be useful? Components would be react-native first then.
Also see:
* https://github.com/jaredpalmer/formik
* https://github.com/nachos-ui/nachos-ui 
* https://github.com/kiwicom/universal-components
* https://github.com/Microsoft/reactxp/

PS Can you squash your commits?

**Comment by jeluard** [comment]
@eswarasai Note that you have to apply to each individual tasks so that you can be approved.

**Comment by jeluard** [comment]
Thanks @eswarasai !

**Comment by eswarasai** [comment]
@jeluard -- Thanks for the quick review and also helpful links for setting up storybook for React-Native. I'll go through them and share a viable option for a monorepo setup.

**Pull request merged**